plugins/c8y_log_plugin/src/main.rs

```diff
@@ -104,44 +109,10 @@ async fn run(
         tokio::select! {
                 message = mqtt_client.received.next() => {
                 if let Some(message) = message {
-                    if is_c8y_bridge_up(&message) {
-                        plugin_config = read_log_config(config_file);
-                        let () = handle_dynamic_log_type_update(&plugin_config, mqtt_client).await?;
-                    }
-                    if let Ok(payload) = message.payload_str() {
-                        let result = match payload.split(',').next().unwrap_or_default() {
-                            "522" => {
-                                info!("Log request received: {payload}");
-                                // retrieve smartrest object from payload
-                                let maybe_smartrest_obj = SmartRestLogRequest::from_smartrest(payload);
-                                if let Ok(smartrest_obj) = maybe_smartrest_obj {
-                                    handle_logfile_request_operation(
-                                        &smartrest_obj,
-                                        &plugin_config,
-                                        mqtt_client,
-                                        http_client,
-                                    )
-                                    .await
-                                } else {
-                                    error!("Incorrect SmartREST payload: {}", payload);
-                                    Ok(())
-                                }
-                            }
-                            _ => {
-                                // Ignore operation messages not meant for this plugin
-                                Ok(())
-                            }
-                        };
-
-                        if let Err(err) = result {
-                            let error_message = format!("Handling of operation: '{}' failed with {}", payload, err);
-                            error!("{}", error_message);
-                        }
-                    }
-                }
-                else {
+                    process_mqtt_messages(message, &plugin_config, mqtt_client, http_client, config_file).await?;
```

Comment:
Very small detail. The function `process_mqtt_messages` accepts always a single message, so the function name should be `process_mqtt_message`, without 's'.

```suggestion
                    process_mqtt_message(message, &plugin_config, mqtt_client, http_client, config_file).await?;
```

Reply:
Yes, that's right. Fixed it.